libraries/wallet/wallet.cpp

```diff
@@ -1269,6 +1269,29 @@ class wallet_api_impl
       return sign_transaction( tx, broadcast );
    } FC_CAPTURE_AND_RETHROW( (from)(symbol)(amount)(broadcast) ) }
 
+   signed_transaction claim_asset_fee_pool(string symbol,
+                                           string amount,
+                                           bool broadcast /* = false */)
+   { try {
+      optional<asset_object> asset_to_fund = find_asset(symbol);
```

Comment:
asset_to_fund is not a good name, I suppose that's copy&pasted from fund_fee_pool...